### DISCUSSION TITLE
Is it possible to parse the webview content?

### DISCUSSION THREAD

**Discussion by bachrc** [tauri-apps/wry#1242] [Q&A]
Hello everyone! I was wondering about a tauri app, where we could visit some external webpages. (github, hacker news..)

The user would navigate on the web in the tauri/wry app, and at some time, I would like to parse the inner html in the wry webview. I searched in both docs and API, and I can't see how I could get the rendered HTML content in the Rust code. Do you have any idea? Thanks!

**Comment by ImageDeeply** [comment]
Maybe I'm missing some subtle point ... but I think the answer is to get in JavaScript document.documentElement.outerHTML or similar, then call that from Rust.

**Comment by ilyazub** [comment]
Here's the code to render HTML and output to stdout: https://github.com/tauri-apps/wry/discussions/761#discussioncomment-7579658
